src/components/List.js

```diff
@@ -1,12 +1,54 @@
 import React from 'react';
 import useFirestore from '../hooks/useFirestore';
+import { db } from '../lib/firebase';
 
 const List = ({ token }) => {
   const { docs } = useFirestore(token);
+
+  const checkPurchasedDate = (purchasedDate) => {
+    if (purchasedDate === null) {
+      return false;
+    }
+    const today = new Date();
+    let daySincePurchased = purchasedDate.toDate();
+
+    daySincePurchased.setDate(daySincePurchased.getDate() + 1); // Add one day to the last purchased date
+
+    // If today > daySincePurchased at least 24 hours have passed, return false to uncheck box
+    if (
```

Comment:
Nit-pick: 
Any time you find yourself with a conditional or ternary, in which you are returning just `true` or `false`, you can actually just return the conditional statement itself. If you're returning false if the conditional is true, as is the case here, then you can then negate the conditional statement and return it.

Reply:
Hi Mike. I've simplified the code a bit from the comments Chris made above. When I have this `return (today > daySincePurchased);` I'm unable to check the boxes. Not sure why this is happening. Your suggestion makes sense to me

Without a `return true;` I get this error
![Screen Shot 2021-01-29 at 6 10 03 PM](https://user-images.githubusercontent.com/14856004/106339835-63285400-625d-11eb-9745-50fa5631edb4.png)

Reply:
I believe that this error indicates that you are returning undefined somehow. when you console.log(today > daySincePurchased), what does it evaluate to? Are you returning early anywhere in the function?

Reply:
I believe that you need to switch it to `daySincePurchased > today`.

Reply:
So much fun! 🤯 I need to think on this before I push any changes 😃

Reply:
I believe we wrote it that way because we were getting confused with the logic and wanted to make it easier for our brains to understand what was going on 😅